cpp/src/IceGrid/SessionI.cpp

```diff
@@ -63,6 +63,24 @@ BaseSessionI::BaseSessionI(const string& id, const string& prefix, const shared_
     }
 }
 
+void
+BaseSessionI::keepAlive(const Ice::Current& current)
+{
+    lock_guard lock(_mutex);
+    if (_destroyed)
+    {
+        throw Ice::ObjectNotExistException(__FILE__, __LINE__, current.id, "", "");
```

Comment:
Is this the correct way to throw ONE? I thought we didn't need to set the id.

Reply:
That's indeed incorrect.

Reply:
Fixed